CMake/External_OpenCV.cmake

```diff
@@ -197,6 +222,7 @@ ExternalProject_Add(OpenCV
     -DBUILD_SHARED_LIBS:BOOL=True
     -DBUILD_TESTS:BOOL=False
     -DWITH_EIGEN:BOOL=${fletch_ENABLE_EIGEN}
+    -DWITH_JASPER:BOOL=False
```

Comment:
Right, I was suggesting we turn it off in both cases, not just when not enabling CUDA. Not a showstopper though.